docroot/modules/custom/foia_webform/src/FoiaSubmissionServiceApi.php

```diff
@@ -336,6 +337,26 @@ protected function replaceFidsWithFileContents(WebformInterface $webform, array
     }
   }
 
+  /**
+   * Add reference to files that have been removed by virus scanning.
+   *
+   * @param \Drupal\webform\WebformInterface $webform
+   *   The webform the submission belongs to.
+   * @param array $submissionValues
+   *   The values submitted.
+   */
+  protected function includeReferenceToRemovedFiles(WebformInterface $webform, array &$submissionValues) {
+    $fileAttachmentElementNames = $this->getFileAttachmentElementsOnWebform($webform);
+    if ($fileAttachmentElementNames) {
+      foreach ($fileAttachmentElementNames as $fileAttachmentElementName) {
+        if (isset($submissionValues[$fileAttachmentElementName])) {
+          $attachments = $this->getRemovedFiles($submissionValues[$fileAttachmentElementName]);
```

Comment:
It looks like fileAttachmentElementName an array of fids below. Should the name of the variable be changed accordingly?

Reply:
`$fileAttachmentElementName` is the array key for the file field, for example `attachments_supporting_documentation`.  `$submissionValues[$fileAttachmentElementName]` should be an array of fids.